spec/acceptance/windows_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance_windows' if Gem.win_platform?
+require 'json'
+
+describe 'sensu::agent class', if: Gem.win_platform? do
+  context 'default' do
+    pp = <<-EOS
+    class { '::sensu': }
+    class { '::sensu::agent':
+      package_source => 'https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.7.0/sensu-go-agent_5.7.0.2380_en-US.x64.msi',
+      backends       => ['sensu_backend:8081'],
+      config_hash    => {
+        'name' => 'sensu_agent',
+      }
+    }
+    EOS
+
+    File.open('C:\manifest-agent.pp', 'w') { |f| f.write(pp) }
```

Comment:
ouch, no easier way on windows?

Reply:
This is what we had to do with sensuclassic.  We can't use beaker tools as they expect to connect to something and don't work by running local commands.  The module to run local beaker stuff never seemed to work so was just easier to use regular Ruby.

Reply:
I tried to use `beaker-windows` but the functions expect to be used with `hosts` or `host` objects from beaker and those would expect something like winrm and not just local commands.